lua/wire/client/text_editor/modes/e2.lua

```diff
@@ -333,6 +333,9 @@ function EDITOR:SyntaxColorLine(row)
     local spaces = self:SkipPattern( " *" )
     if spaces then addToken( "comment", spaces ) end
 
+    local invalidInput = self:SkipPattern( "^[^A-Z:%[]" )
```

Comment:
You don't need the `^` considering `SkipPattern` checks this internally. I'd add a `*` so it doesn't take multiple iterations to skip noise.

Other than the above I think this'll work, I'll do a few more tests.